qulice-checkstyle/src/test/java/com/qulice/checkstyle/CheckstyleValidatorTest.java

```diff
@@ -698,7 +698,7 @@ public void rejectsNonDiamondOperatorUsage() throws Exception {
      * CheckstyleValidator can allow diamond operator usage.
      * @throws Exception If error
      * @todo #715:30min add test for next situation
-     *  `return new ArrayList<String>();`
+     *  `@code{return new ArrayList&lt;String&gt;();}`
```

Comment:
@vkuchyn currently it is incorrect, please replace it with proper `@code` tag and put `<`, `>` without escapes